service-rpc/src/main/java/org/apache/iotdb/rpc/IoTDBRpcDataSet.java

```diff
@@ -312,9 +317,23 @@ public float getFloat(String columnName) throws StatementExecutionException {
     checkRecord();
     int index = columnOrdinalMap.get(columnName) - START_INDEX;
     if (!isNull(index, rowsIndex - 1)) {
+      lastReadWasNull = false;
       return BytesUtils.bytesToFloat(values[index]);
     } else {
-      throw new StatementExecutionException(String.format(VALUE_IS_NULL, columnName));
+      lastReadWasNull = true;
+      return 0;
+    }
+  }
+
+  public short getShort(String columnName) throws StatementExecutionException {
+    checkRecord();
```

Comment:
Actually, we are not supporting short type of data yet...

Reply:
I think so, too. I felt weird that why it was implemented. How about just throwing exception while it has been used?

Reply:
Yes, look good to me. 👍